DependencyInjection/Configuration.php

```diff
@@ -79,6 +79,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('title')->end()
                 ->scalarNode('description')->end()
                 ->scalarNode('original_route_pattern')->defaultValue(SeoPresentation::ORIGINAL_URL_CANONICAL)->end()
+                ->scalarNode('locale')->defaultNull()->end()
```

Comment:
I do not see how to get the current locale now? Or what do we configure here? The key for the locale parameter in the routing or one of the locales?